projects/plugins/crm/.gitattributes

```diff
@@ -9,6 +9,7 @@
 /vendor/composer/**                             production-include
 /vendor/jetpack-autoloader/**                   production-include
 /vendor/dompdf/**                               production-include
+/vendor/fortawesome/font-awesome/**             production-include
```

Comment:
I wonder if it'd be useful to include only the bits we need, as it looks like there's a lot extra included in the package (less files, scss files, lots of examples, etc).

Reply:
Good idea: 65f61483b3

Reply:
Good idea: 65f61483b3

Reply:
Good idea: 65f61483b3